nodes/src/nodes/llm_vision_openai/IInstance.py

```
@@ -0,0 +1,132 @@
+                question = Question()
+                image_base64 = base64.b64encode(bytes(self.image_data)).decode('utf-8')
+                image_data_url = f'data:{mimeType};base64,{image_base64}'
+                question.addContext(image_data_url)
+                question.addQuestion(self.IGlobal._chat._prompt)
+
+                try:
+                    answer = self.IGlobal._chat.chat(question)
+                    self._cached_answer = answer.getText()
+                    self._cache_from_image = True
+                    self.instance.writeText(self._cached_answer)
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)